homeworks/vlad.syniakov_Revy-Lagoon09/homework_5/3.js

```diff
@@ -0,0 +1,13 @@
+const Obj = {
+    b: 'c',
+    d: { e: 'f' },
+};
+function innerCopy(mainObj) {
+    const copyObj = {};
+    for ( const key in mainObj) {
+        copyObj[key] = mainObj[key];
```

Comment:
there is nothing about `for...in`, the problem that simply assigning one object to another objects property is not a copy of object, it is actually a copy of kind of "link" to initial object, that's why it's not what wanted